packages/react-reconciler/src/__tests__/ReactLazy-test.internal.js

```diff
@@ -792,6 +792,7 @@ describe('ReactLazy', () => {
     Add.defaultProps = {
       innerWithDefault: 42,
     };
+    Add.displayName = 'Add';
```

Comment:
Let's leave these tests as-is and write a separate one that tests `displayName`.

Reply:
This is only there because the test function matches the display name. Since not every component is a memo component the test would fail.

Reply:
i.e. this is not there to test `displayName` but to prevent refactoring the test.

Reply:
The alternative would be to use a wildcard matcher in https://github.com/facebook/react/blob/4aa22411f49d833589f38271ec4a4d800903114d/packages/react-reconciler/src/__tests__/ReactLazy-test.internal.js#L681
instead of expecting an exact component name.

Reply:
> Let's leave these tests as-is and write a separate one that tests displayName.

Added an explicit test for memo and displayName in 012af538b6b532a8e2403c373b178e360e0f6791.

I could not use displayName in this test but then I would have to change the test to match `*` as the displayName (some components have the memo wrapper some don't). I don't think that's a good idea.

Not sure what else I should do here.